ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Page.cs

```diff
@@ -47,6 +32,8 @@ private Page(string name, string url)
         public static readonly Page EquipmentBlocked = new Page("Equipment Blocked", "equipment-blocked");
 
         // Shared
+        public static readonly Page CheckingVideoHearing = new Page("Checking Video Hearing", "checking-video-hearing");
+
```

Comment:
Is this a shared page or just Rep? You have only added a test for Rep if so?

Reply:
It's shared page.